pgmpy/factors/CPD.py

```diff
@@ -110,9 +110,12 @@ def __init__(self, variable, variable_card, values,
                                     " of strings.")
             self.evidence = evidence
             variables.extend(evidence[::-1])
-            if not len(evidence_card) == len(evidence):
+            if len(evidence_card) < len(evidence):
```

Comment:
Okay. I will change it to a more clear error message. Any suggestions for the error message?